sites/all/modules/ar/ar_assessments/js/ar_assessments.js

```diff
@@ -405,7 +405,7 @@ Drupal.behaviors.arAssessmentsAssessments = {
       },
 
       defaultRoute: function (actions) {
-        this.navigate('list/1', {trigger: true});
```

Comment:
@guillaumev it was decided on https://humanitarian.atlassian.net/browse/HRINFO-594 to try just showing the table view to make it consistent with the other pages so I've changed the default route and removed the tabs.